bugwarrior/services/logseq.py

```diff
@@ -0,0 +1,301 @@
+import logging
+
+import requests
+import typing_extensions
+
+import re
+from datetime import datetime
+
+from bugwarrior import config
+from bugwarrior.services import IssueService, Issue, ServiceClient
+
+log = logging.getLogger(__name__)
+
+
+class LogseqConfig(config.ServiceConfig):
+    service: typing_extensions.Literal["logseq"]
+    host: str = "localhost"
+    port: int = 12315
+    token: str = ""
+    task_state: str = "DOING, TODO, NOW, LATER, IN-PROGRESS, WAIT, WAITING"
+    char_open_link: str = "【"
+    char_close_link: str = "】"
+    char_open_bracket: str = "〈"
+    char_close_bracket: str = "〉"
+    inline_links = True  # set to False to override bugwarrior config
```

Comment:
```suggestion
    inline_links: bool = True
```

May as well annotate the type for consistency. The comment seems superfluous.